### ISSUE TITLE
AVT 1 - React Header Base w/Navigation or the Header Base w/Navigation and Actions UI Shell has DAP violations

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#3572]
## Environment
macOS Mojave version 10.14.5
Chrome Version 75.0.3770.100 (Official Build) (64-bit)
Carbon v10 - React
DAP - July 2019 Ruleset

## Detailed Description
Run DAP on the Header Base w/Navigation or the Header Base w/Navigation and Actions UI Shell. The following 5 violations are identified (see screenshot below):
![HeaderNav](https://user-images.githubusercontent.com/21676914/62078189-a097c580-b211-11e9-8ff8-321e2ab33062.png)

Note: 
1. The contrast issue relates to the Skip to Main link, which fails the WCAG Tests for [G1: Adding a link at the top of each page that goes directly to the main content area](https://www.w3.org/TR/WCAG20-TECHS/G1.html#G1-examples)
2. Add a role to the parent  of the menuitems. Use role="menu" or "menubar" or "group"  to fix the issue. DAP only reports this issue if the page size reduced and the links do not display across the top of the page.
